src/components/DpResettableInput/DpResettableInput.vue

```diff
@@ -13,23 +13,25 @@
       @input="onInput"
       @enter="$emit('enter', currentValue)"
       @focus="$emit('focus')" />
-    <button
-      v-if="!inputAttributes.disabled"
-      class="btn--blank o-link--default pr-0.5"
-      data-cy="resetButton"
-      :class="buttonClass"
-      :disabled="currentValue === defaultValue"
-      @click="resetValue">
-      <dp-icon
-        icon="xmark"
-        :size="iconSize" />
-    </button>
-    <!-- Slot for additional buttons -->
-    <slot />
+    <span class="space-x-0.5">
+      <button
+        v-if="!inputAttributes.disabled"
+        class="btn--blank o-link--default"
+        data-cy="resetButton"
+        :class="buttonClass"
```

Comment:
nitpick: `:class` should be above `data-cy`

Reply:
1bbba2a50ddc4d81e1f317858f4c8eed41baefaa